ui/core/components/filters_menu.tsx

```diff
@@ -1,10 +1,13 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { element } from 'tsx-vanilla';
+
 import { Player } from '../player.js';
 import { ItemSlot } from '../proto/common.js';
 import { RaidFilterOption, SourceFilterOption, UIItem_FactionRestriction } from '../proto/ui.js';
 import { armorTypeNames, raidNames, rangedWeaponTypeNames, sourceNames, weaponTypeNames } from '../proto_utils/names.js';
 import { Sim } from '../sim.js';
 import { EventID } from '../typed_event.js';
-import { BaseModal } from './base_modal.js';
```

Comment:
Actually all these imports can be done without extensions, I've tried to remove them in files I touch